client/lib/analytics/ad-tracking/setup.js

```diff
@@ -49,7 +50,9 @@ if ( typeof window !== 'undefined' ) {
 
 	// Linkedin
 	if ( mayWeInitTracker( 'linkedin' ) ) {
-		setupLinkedinInsight( isJetpackCloud() ? TRACKING_IDS.jetpackLinkedinId : null );
+		setupLinkedinInsight(
+			isJetpackCloud() || isJetpackCheckout() ? TRACKING_IDS.jetpackLinkedinId : null
```

Comment:
Minor improvement, I could see a situation in which we'd add linkedin tracking to more pages. I think it'd be nice to store them in an array of booleans like
```js
// variable name could use some work
const shouldSetupLinkedIn = [
   isJetpackCloud(),
   isJetpackCheckout(),
   ...
];
```

And then just check if any items are true here

Reply:
The thing is, there are only either Jetpack or Wpcom-related pages in Calypso. So it will be either these conditions (if we want to include something on Jetpack), the negation (if we want to include something on Calypso), or none if it should be everywhere.

It's also the thing that we will not implement the same tracking id for jetpack and calypso - so we will most probably not add any more conditions to this array.